indra/llwindow/llgamecontrol.cpp

```diff
@@ -692,7 +847,7 @@ void LLGameControllerManager::getFlycamInputs(std::vector<F32>& inputs)
         // value arrives as S16 in range [-32768, 32767]
         // so we scale positive and negative values by slightly different factors
         // to try to map it to [-1, 1]
-        F32 input = F32(axis) * ((axis > 0.0f) ? 3.051850476e-5 : 3.0517578125e-5f) * channel.mSign;
+        F32 input = F32(axis) / ((axis > 0.0f) ? 32767 : 32768) * channel.mSign;
```

Comment:
While compilers are much smarter these days, we are working with a multiple platforms, multiplication is likely faster at least somewhere.

Reply:
> multiplication is likely faster at least somewhere

Agree, but this can be felt when CPU does millions of divisions per second
Regarding this case, using 32768 and 32767 looks more logical and easy to check
E.g., any mistake in the floating point value won't be noticed but could be critical

Reply:
> any mistake in the floating point value won't be noticed but could be critical

I don't think we need that much of a precision and you can declare some `const F32 min = 1.f/32768.f;` to make sure division happens once.